requirements.txt

```
@@ -5,6 +5,7 @@
 #  when using a custom variant like `vtk_osmesa`
 #  which we do in some of our CI jobs.
 imageio<2.27.0
+matplotlib>=3.5.0,<3.7.2
```

Adding `>=3.5.0` here ties in with my earlier comment about this.

---

I'll remove it ref:https://github.com/pyvista/pyvista/pull/4156#issuecomment-1476304697 .
```suggestion
matplotlib<3.7.2
```